include/micm/solver/rosenbrock.inl

```diff
@@ -240,26 +240,21 @@ namespace micm
   {
     MICM_PROFILE_FUNCTION();
 
-    auto jacobian = state.jacobian_;
     uint64_t n_consecutive = 0;
     singular = false;
     while (true)
     {
+      auto jacobian = state.jacobian_;
```

Comment:
Does `jacobian` need copy at every iteration? Does it mean we did wrong previously?

Reply:
Yes, the `jacobian` needs to be copied every iteration because `alpha` is changed due to the time-step refinement. The previous implementation was correct because we never triggered this iteration about time-step refinement. Thus it did not matter where the copy happened.

Reply:
Interesting, what is time step refinement?

Reply:
This line: https://github.com/NCAR/micm/blob/fc66c175a5a7857a7e2c76cf1726d6e55855b8c3/include/micm/solver/rosenbrock.inl#L261

Reply:
ah, thanks!